Ginger/Ginger/ALM/ALMConnectionPage.xaml.cs

```diff
@@ -59,6 +60,14 @@ public ALMConnectionPage(ALMIntegration.eALMConnectType almConnectStyle, bool is
                     App.UserProfile.Solution.AlmType = ALMIntegration.eALMType.QC;
             }
 
+            if (!WorkSpace.Instance.BetaFeatures.RestAPI)
+            {
+                RestAPICheckBox.Visibility = Visibility.Hidden;
+                App.UserProfile.Solution.UseRest = false;
+                RestAPICheckBox.IsChecked = false;
+                RestAPICheckBox.IsEnabled = false;
+            }
+
             if (almConnectStyle != ALMIntegration.eALMConnectType.Silence)
```

Comment:
if only Attachment left so you can remove the Beta flag, it can be visible to users now